apps/web/modules/ui/components/no-code-action-form/components/page-url-selector.test.tsx

```
@@ -95,25 +105,41 @@ vi.mock("@/modules/ui/components/button", () => ({
+        </SelectContext.Provider>
+      );
+    },
+    SelectContent: ({ children }: any) => <div data-testid="select-content">{children}</div>,
+    SelectItem: ({ children, value }: any) => {
+      const context = React.useContext(SelectContext);
+      return (
+        <div // NOSONAR // This is a mocked component to test the logic
+          data-testid={`select-item-${value}`}
+          data-value={value}
+          onClick={() => context.onValueChange?.(value)}
+          style={{ cursor: "pointer" }}>
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)